tests/eko/test_couplings.py

```diff
@@ -257,8 +254,7 @@ def benchmark_expanded_n3lo(self):
         m2c = 2
         m2b = 25
         m2t = 30625
-        threshold_list = [m2c, m2b, m2t]
-        threshs = MatchingScales.from_dict(dict(zip("cbt", threshold_list)))
+        threshold_list = [m2c / masses[0], m2b / masses[1], m2t / masses[2]]
```

Comment:
@giacomomagni I wonder if this was another bug ... note that in the old implementation `threshs` were absolute values (i.e. [GeV^2]) where as we always expect ratios (look at the argument name) - the bug fixed in this PR here is just that the ratio of the arguments should be between squared values

Reply:
Note that I lowered the `rtol` in the test by almost an order of magnitude, though I wonder whether those two things are correlated ...

Reply:
The run is between 90 GeV and 100 GeV. Simply it was not crossing any threshold before, and it is not doing it even now (moreover, since the ratio was squared, it is running in both cases in 5 flavors).